splunk-otel-android/src/main/java/com/splunk/rum/CrashReporter.java

```diff
@@ -50,12 +50,13 @@ static void initializeCrashReporting(Tracer tracer, OpenTelemetrySdk openTelemet
         public void uncaughtException(@NonNull Thread t, @NonNull Throwable e) {
             StringWriter writer = new StringWriter();
             e.printStackTrace(new PrintWriter(writer));
-            tracer.spanBuilder("crash.report")
+            String exceptionType = e.getClass().getSimpleName();
+            tracer.spanBuilder(exceptionType)
                     .setAttribute(SemanticAttributes.THREAD_ID, t.getId())
                     .setAttribute(SemanticAttributes.THREAD_NAME, t.getName())
                     .setAttribute(SemanticAttributes.EXCEPTION_STACKTRACE, writer.toString())
                     .setAttribute(SemanticAttributes.EXCEPTION_ESCAPED, true)
-                    .setAttribute(SplunkRum.COMPONENT_KEY, SplunkRum.COMPONENT_ERROR)
+                    .setAttribute(SplunkRum.COMPONENT_KEY, SplunkRum.COMPONENT_CRASH)
```

Comment:
I notice that `COMPONENT_ERROR` is still used for the ANRs...is that intentional?

Reply:
Hmm, I guess so? ANRs are not necessarily fatal, unrecoverable errors, it's just the application being horribly slow.
(At the very least, that's my understanding - and I don't know that much about Android 😅 )